builder/amazon/common/ami_config.go

```diff
@@ -170,26 +170,33 @@ func (c *AMIConfig) Prepare(accessConfig *AccessConfig, ctx *interpolate.Context
 		}
 	}
 
-	var kmsKeys []string
+	kmsKeys := make([]string, 0)
```

Comment:
Nit: this would be the same as the simpler form :
```suggestion
	kmsKeys := make([]string)
```

Reply:
uh, merged this and it broke: `builder/amazon/common/ami_config.go:173:17: missing len argument to make([]string)`... reverting